llama-index-core/llama_index/core/base/query_pipeline/query.py

```diff
@@ -107,7 +107,7 @@ class OutputKeys(BaseModel):
     def from_keys(
         cls,
         required_keys: Set[str],
-    ) -> "InputKeys":
+    ) -> "OutputKeys":
         """Create InputKeys from tuple."""
```

Comment:
NIT: we probably should also update the doc string?

Reply:
ehh all good -- i got us covered :)

Reply:
@nerdai Thank you